hw/xilinx/README.md

```diff
@@ -7,7 +7,7 @@ More technical documentation can be found in the `doc/` folder for:
 * [Vivado-specific patches](doc/VIVADO_PATCHES.md).
 
 ## Build Bitstream and Program Device
-To build the bitstream just run:
+Once rtl files are fetched in `hw/units`, to build the bitstream just run:
```

Comment:
Just like it's done here, without back-referencing the root Makefie, I would add a general note about building IPs (xilinx and custom) and the need for all sources to be ready for custom IPs -> see hw/units/README.md.